entrypoint.sh

```diff
@@ -68,15 +70,22 @@ changelog=${changelog:-false}
 preset=${preset:-conventionalcommits}
 outputtype=${outputtype:-stdout}
 repopath=${repopath:-./}
+bumpoverride=${bumpoverride:-}
+
 
 # verify git repository
 if [ ! $(git rev-parse --is-inside-work-tree 2>&1) ]; then
   echo "volume is not a git repository workspace"
   exit 1
 fi
 
+if [[ $bumpoverride != "" && $bumpoverride != "major" && $bumpoverride != "minor" && $bumpoverride != "patch" ]]; then
+  echo "Unknown bump override level, possible values: 'major', 'minor' or 'patch'"
+  exit 1
+fi
+
 # step into repopath
-cd $repopath
+cd "$repopath" || exit
```

Comment:
why is this `exit` meant to do here?

Reply:
This was autofixed by vscode (through extension and shellcheck)
Relevant shellcheck entry: https://www.shellcheck.net/wiki/SC2164

> cd can fail for a variety of reasons: misspelled paths, missing directories, missing permissions, broken symlinks and more.
> If/when it does, the script will keep going and do all its operations in the wrong directory. This can be messy, especially if the operations involve creating or deleting a lot of files.
> To avoid this, make sure you handle the cases when cd fails. Ways to do this include
> cd foo || exit as suggested to abort immediately, reusing exit code from failed cd command
> cd foo || { echo "Failure"; exit 1; } abort with custom message
> cd foo || ! echo "Failure" omitting "abort with custom message"
> if cd foo; then echo "Ok"; else echo "Fail"; fi for custom handling
<(cd foo && cmd) as an alternative to <(cd foo || exit; cmd) in <(..), $(..) or ( )

Reply:
yeah, but we don't want the app exiting at this point,
an error thrown will be much more useful.

you can [ignore](https://github.com/koalaman/shellcheck/wiki/Directive) this specific error,
add `# shellcheck disable=SC2164` above that statement.

Reply:
BTW
is the `shellcheck` extension the one responsible for adding all of these quotation marks?

because I seem to remember an issue I had with them, but I don't remember what it was.

anyway, we use [shfmt](https://github.com/mvdan/sh) for linthing shell scripts for this repository, so maybe it's best if you drop the `shellcheck` vscode extension for this specific project and revert back all the modifications it made?
it looks like it doesn't see eye-to-eye with `shfmt`.